packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -129,6 +129,10 @@ export default function () {
   return {
     inherits: require("babel-plugin-transform-strict-mode"),
 
+    manipulateOptions(opts, parserOpts) {
+      parserOpts.plugins.push("objectRestSpread");
+    },
```

Comment:
It seems there are no merging util for configuration. I was thinking to use `Object.assign`.